test/Mono.Linker.Tests.Cases/Mono.Linker.Tests.Cases.csproj

```diff
@@ -38,84 +38,14 @@
     <Content Include="TypeForwarding\Dependencies\TypeForwarderMissingReference.il" />
     <Content Include="Attributes\OnlyKeepUsed\Dependencies\AssemblyWithUnusedAttributeOnReturnParameterDefinition.il" />
     <Content Include="Attributes\OnlyKeepUsed\Dependencies\AssemblyWithUnusedAttributeOnGenericParameter.il" />
-    <Content Include="Attributes\OnlyKeepUsed\UnusedAttributePreservedViaLinkXmlIsKept.xml" />
```

Comment:
Why do we remove all these xml files?

Reply:
They seemed to be necessary only to tell the linker which XML's were available. The BodySubstituter and the CustomAttributes steps don't include their xml's in the Test.Cases.csproj and the test now seemed to not use them